modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-okhttp/infrastructure/ApiResponse.kt.mustache

```diff
@@ -6,7 +6,7 @@ package {{packageName}}.infrastructure
 
 {{#nonPublicApi}}internal {{/nonPublicApi}}interface Response
 
-{{#nonPublicApi}}internal {{/nonPublicApi}}abstract class ApiInfrastructureResponse<T>(val responseType: ResponseType): Response {
+{{#nonPublicApi}}internal {{/nonPublicApi}}abstract class ApiResponse<T>(val responseType: ResponseType): Response {
```

Comment:
@wing328 to make the transition easier, could you please create a TypeAlias like this?
```
@Deprecated("Renamed to ApiResponse", ReplaceWith("ApiResponse"))
 {{#nonPublicApi}}internal {{/nonPublicApi}}typealias ApiInfrastructureResponse<T> = ApiResponse<T>
```

This way the users will have a warning in the IDE that will help them to migrate the code

Reply:
Thanks for the tips. I thought about that too. The reason I didn't do it is that the change related to ApiInfrastructureResponse has not been included in the stable release if i'm not mistaken.

Reply:
I think you are right 👍